### PR TITLE
Add farsi to CSF language list

### PR DESCRIPTION
**Pull request by dmcavoy**
Melika asked that farsi be added to the list of supported languages on code.org/csf. I also noticed that English seemed to be missing from the list

### CODE REVIEW

File: pegasus/sites.v3/code.org/public/curriculum/csf.haml [hunk 1]
```diff
@@ -37,7 +37,7 @@ theme: responsive_full_width
     .clear
 
 %section.bg-neutral-light
-  - languages = "العربية, Bahasa Indonesia, Català, 简体字, 繁體字, Čeština, Français, Deutsch, हिन्दी, Italiano, 日本語, 한국어, ಕನ್ನಡ, بهاس ملايو, Монгол хэл, Polski, Português (Brasil), Română, Русский, Slovenčina, Filipino, தமிழ், ภาษาไทย, Türkçe, Українська, Español (España), Español (LATAM), اردو, O'zbekcha, Tiếng Việt"
+  - languages = "فارسی ,العربية , Bahasa Indonesia, Català, 简体字, 繁體字, Čeština, Français, Deutsch, हिन्दी, Italiano, 日本語, 한국어, ಕನ್ನಡ, بهاس ملايو, Монгол хэл, English, Polski, Português (Brasil), Română, Русский, Slovenčina, Filipino, தமிழ், ภาษาไทย, Türkçe, Українська, Español (España), Español (LATAM), اردو, O'zbekcha, Tiếng Việt"
```
Comment (bethanyaconnor) [inline pegasus/sites.v3/code.org/public/curriculum/csf.haml:40]: I checked with @castro-jorge in the past and he said English didn't need to be added here.

Comment (dmcavoy) [reply pegasus/sites.v3/code.org/public/curriculum/csf.haml:40]: I think we should list all the languages including English

Comment (kelbyhawn) [reply pegasus/sites.v3/code.org/public/curriculum/csf.haml:40]: I agree re: including English.

**Comment by dmcavoy** [comment]
I'm going to merge this

**Pull request merged**